src/cli/commands/create/app.ts

```diff
@@ -120,6 +120,12 @@ export async function runBootstrapper(
       isAnonymous: false,
     },
     packageManagerType: packageManagerType,
+    packageJson: {
+      // todo
```

Comment:
todo what?

Reply:
The `content` is set to `{}`, we might need it in the future during init which might break the workflow